### PR TITLE
Remove dependence on external mysql image tag order

### LINKED ISSUE #11381: oc get istag/mysql:5.5 fails
Not sure if this is a flake or something more fundamental, it is failing consistently since friday night.

https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin_check/7166/consoleFull#87565479056c25df4e4b05b79524e5d52

```
FAILURE after 59.513s: test/cmd/images_tests.sh:177: executing 'oc get istag/mysql:5.5' expecting success; re-trying every 0.2s until completion or 60.000s: the command timed out
```

i've confirmed that mysql:5.5 can be imported/imagestream created on my machine.

### PR DESCRIPTION
**Pull request by liggitt**
Resolves #11381

### CODE REVIEW

**Comment by miminar** [comment]
LGTM [test]

**Comment by legionus** [comment]
I think it fix https://github.com/openshift/origin/issues/10928 too.

**Comment by miminar** [comment]
> I think it fix #10928 too.

I don't think so, we still import from the same location (docker hub). The timeout can occur again.

**Comment by legionus** [comment]
> I don't think so, we still import from the same location (docker hub). The timeout can occur again.

@miminar Yes, I was wrong.

**Comment by miminar** [comment]
@liggitt good job! Jenkins just passed image cmd images test.

**Comment by liggitt** [comment]
[merge]

File: test/cmd/images_tests.sh [hunk 1]
```diff
@@ -197,7 +196,7 @@ os::cmd::expect_success_and_text "oc get is/reference --template='{{(index .spec
 os::cmd::expect_success 'oc new-project test-cmd-images-2'
 os::cmd::expect_success "oc tag $project/mysql:5.5 newrepo:latest"
 os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}'" 'ImageStreamImage'
-os::cmd::expect_success_and_text 'oc get istag/newrepo:latest -o jsonpath={.image.dockerImageReference}' 'mysql@sha256:'
+os::cmd::expect_success_and_text 'oc get istag/newrepo:latest -o jsonpath={.image.dockerImageReference}' 'openshift/mysql-55-centos7@sha256:'
```
Comment (bparees) [inline test/cmd/images_tests.sh]: it'd be better to use centos/mysql-56-centos7 since it'll be around longer.

Comment (liggitt) [inline test/cmd/images_tests.sh]: sorry, didn't see this until post-merge. @miminar, can you follow up with any additional improvements to lengthen the lifespan of the test?

File: test/cmd/images_tests.sh [hunk 2]
```diff
@@ -128,16 +128,15 @@ os::cmd::expect_success_and_text 'oc import-image mysql:latest' "sha256:"
...
 os::cmd::expect_success_and_text "oc get istag/mysql:external --template='{{.tag.from.kind}}'" 'DockerImage'
 os::cmd::expect_success_and_text "oc get istag/mysql:external --template='{{.tag.from.name}}'" 'docker.io/mysql'
-os::cmd::expect_success 'oc delete is/mysql'
 # import creates new image stream with single tag
-os::cmd::expect_failure_and_text 'oc import-image mysql:latest --from=docker.io/mysql:latest' '\-\-confirm'
-os::cmd::expect_success_and_text 'oc import-image mysql:latest --from=docker.io/mysql:latest --confirm' 'sha256:'
-os::cmd::expect_success_and_text "oc get is/mysql --template='{{(len .spec.tags)}}'" '1'
-os::cmd::expect_success 'oc delete is/mysql'
+os::cmd::expect_failure_and_text 'oc import-image mysql-new-single:latest --from=docker.io/mysql:latest' '\-\-confirm'
+os::cmd::expect_success_and_text 'oc import-image mysql-new-single:latest --from=docker.io/mysql:latest --confirm' 'sha256:'
```
Comment (bparees) [inline test/cmd/images_tests.sh]: can we not get rid of all the docker.io/mysql refs in these tests?

File: test/cmd/images_tests.sh (hunk 1 above)
Comment (bparees) [inline test/cmd/images_tests.sh]: yeah i didn't make the comment until post merge :)

File: test/cmd/images_tests.sh (hunk 2 above)
Comment (liggitt) [inline test/cmd/images_tests.sh]: possibly... the point was to test tagging in things from other sources, but there may be a more contained way to do it. @miminar ^

Comment (miminar) [inline test/cmd/images_tests.sh]: on it

File: test/cmd/images_tests.sh (hunk 1 above)
Comment (miminar) [inline test/cmd/images_tests.sh]: sure

**Pull request merged**